dali/core/mm/mm_test_utils.h

```diff
@@ -198,16 +198,17 @@ class test_resource_wrapper<owning, security_check, stream_aware_memory_resource
     }, ptr, bytes, alignment);
   }
 
-  void *do_allocate_async(size_t bytes, size_t alignment, stream_view sv) override {
+  void *do_allocate_async(size_t bytes, size_t alignment, stream_view strm_vw) override {
```

Comment:
Es ist kein Stormvolkswagen ;)
Perhaps just
```suggestion
  void *do_allocate_async(size_t bytes, size_t alignment, stream_view stream) override {
```

Reply:
It follows general convention and indicates type, so I'm leaving it as it is.